lidar.py

```diff
@@ -0,0 +1,86 @@
+import smbus
+import threading
+import time
+import RPi.GPIO as GPIO
+from collections import deque
+
+
+class LIDARLite(threading.Thread):
+
+    ADDR = 0x62
+    I2C_BUS_NO = 1
+    INTERRUPT_GPIO_PIN = 4
+
+    def __init__(self):
+        super().__init__()
+        self.bus = smbus.SMBus(LIDARLite.I2C_BUS_NO)
+        self.address = LIDARLite.ADDR
+        self.since_bias_correction = 0  # used to perform a reciever bias correction every n measurements
+        self.bias_correction_interval = 100
+        self._terminate = threading.Event()
+
+    def run(self):
+        print("LIDAR Lite thread initialized on address %s" % (self.address))
+        self.initialize()
+        while True:
+            if self._terminate.is_set():
+                self.cleanup()
+                break
+
+            GPIO.wait_for_edge(LIDARLite.INTERRUPT_GPIO_PIN, GPIO.FALLING)  # blocks until we see a falling edge on the LIDAR
```

Comment:
use `self.INTERRUPT_GPIO_PIN` to avoid a global lookup in a hot loop

Reply:
Fixed.